### PR TITLE
collectors/freeipmi: add ipmi-sensors function

### PR DESCRIPTION
**Pull request by ilyam8**
##### Summary

Screenshot 2023-11-09 at 10 20 44

##### Test Plan

Make sure the ipmi-sensors function reports correct values.

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

File: collectors/freeipmi.plugin/freeipmi_plugin.c [hunk 1]
```diff
@@ -1422,20 +1439,194 @@ static size_t send_ipmi_sel_metrics_to_netdata(struct netdata_ipmi_state *state)
...
+            return "critical";
+        default:
+            return "normal";
+    }
+}
+
+static void freeimi_function_sensors(const char *transaction, char *function __maybe_unused, int timeout __maybe_unused, bool *cancelled __maybe_unused) {
+    time_t expires = now_realtime_sec() + update_every;
+
+    BUFFER *wb = buffer_create(PLUGINSD_LINE_MAX, NULL);
```
Comment (thiagoftsm) [inline collectors/freeipmi.plugin/freeipmi_plugin.c:1476]: When you worked with `diskspace` plugin, you defined a buffer [type](https://github.com/netdata/netdata/blob/4c867cb3a61c6c2bcd32f82267b0e01327eba752/collectors/diskspace.plugin/plugin_diskspace.c#L641), but here you are not defining it. Is there any reason for this change? I would suggest we proceed with this definition.

Comment (ilyam8) [reply collectors/freeipmi.plugin/freeipmi_plugin.c:1476]: internal vs external plugin

Comment (ilyam8) [reply collectors/freeipmi.plugin/freeipmi_plugin.c:1476]: I see we [set content-type here](https://github.com/ilyam8/netdata/blob/b410b0de808091d38b8a0cb11602661e360152c9/collectors/freeipmi.plugin/freeipmi_plugin.c#L1610) in external plugins

**Review by thiagoftsm — APPROVED** [review]
We do not have compilation warnings and as image shows, PR is working as expected.

**Pull request merged**